contrib/pkg/createcluster/create.go

```diff
@@ -373,11 +431,32 @@ func (o *Options) GenerateObjects() ([]runtime.Object, error) {
 		}
 	}
 
-	result = append(result, cd)
+	result = append(result, installConfigSecret, cd)
```

Comment:
Should the install config secret be excluded when IncludeSecrets is false?

Reply:
Tough one, but looking at that option and what it's for today it feels more like it's global files you use across most of your clusters, whereas the install-config is heavily generated and previous embedded in the cluster deployment. I think it would cause problems for someone using include secrets rather than help them. For now I'd say lets keep it outside that concept.

Reply:
I am not sure what the purpose of the include secrets is, so I will defer to you.